ui/static/ui/js/listing_resources.jsx

```diff
@@ -430,6 +432,7 @@ define('listing_resources', ['react', 'jquery', 'lodash', 'utils'],
     var ListingPage = React.createClass({
       render: function() {
         return <div>
```

Comment:
If we had two `ListingPage` components the id would conflict with each other. If we need to identify a particular DOM node within the React component then we can use `ref` instead.